src/ceph-volume/ceph_volume/devices/lvm/prepare.py

```diff
@@ -23,19 +23,7 @@ def prepare_dmcrypt(key, device, device_type, tags):
         return ''
     tag_name = 'ceph.%s_uuid' % device_type
     uuid = tags[tag_name]
-    # format data device
-    encryption_utils.luks_format(
-        key,
-        device
-    )
-    encryption_utils.luks_open(
-        key,
-        device,
-        uuid
-    )
-
-    return '/dev/mapper/%s' % uuid
-
+    return encryption_utils.prepare_dmcrypt(key, device, uuid)
```

Comment:
@ifed01 can we make use of `encryption_utils.prepare_dmcrypt()` in `ceph-volume/ceph_volume/devices/raw/prepare.py` (mainly for consistency...)

Reply:
done